inc/Engine/Optimization/DeferJS/DeferJS.php

```diff
@@ -289,4 +274,36 @@ public function exclude_jquery_upgrade() {
 
 		update_option( 'wp_rocket_settings', $options );
 	}
+
+	/**
+	 * Get exclusion list pattern.
+	 *
+	 * @return string
+	 */
+	private function get_inline_exclusions_list_pattern() {
+		$inline_exclusions_list = $this->inline_exclusions;
+
+		/**
+		 * Filters the patterns used to find inline JS that should not be deferred
+		 *
+		 * @since 3.8
+		 *
+		 * @param array $inline_exclusions_list Array of inline JS that should not be deferred.
+		 */
+		$additional_inline_exclusions_list = apply_filters( 'rocket_defer_inline_exclusions', null );
```

Comment:
Why do you use `null` on apply_filters? 
`$additional_inline_exclusions_list` should be an array, right?
So why not using the default empty array?

Reply:
as discussed with you on slack, we need to be compatible with both string and array types for backward compatibility.